test/builder/QuarkBuilderMorphoVaultSupply.t.sol

```diff
@@ -0,0 +1,955 @@
+        assertNotEq(result.eip712Data.hashStruct, hex"", "non-empty hashStruct");
+    }
+
+    function testMorphoVaultSupplyWithBridge() public {
+        QuarkBuilder builder = new QuarkBuilder();
+        QuarkBuilder.BuilderResult memory result = builder.morphoVaultSupply(
+            morphoSupplyIntent_(8453, 5e6, "USDC"),
+            chainAccountsList_(6e6), // holding 3 USDC in total across chains 1, 8453
+            paymentUsd_()
+        );
+
+        assertEq(result.paymentCurrency, "usd", "usd currency");
+
+        // Check the quark operations
+        // first operation
+        assertEq(result.quarkOperations.length, 2, "two operations");
+        assertEq(
+            result.quarkOperations[0].scriptAddress,
+            address(
+                uint160(
+                    uint256(
+                        keccak256(
+                            abi.encodePacked(
+                                bytes1(0xff),
+                                /* codeJar address */
+                                address(CodeJarHelper.CODE_JAR_ADDRESS),
+                                uint256(0),
+                                /* script bytecode */
+                                keccak256(type(CCTPBridgeActions).creationCode)
+                            )
+                        )
+                    )
+                )
+            ),
+            "script address is correct given the code jar address on mainnet"
+        );
+        assertEq(
+            result.quarkOperations[0].scriptCalldata,
+            abi.encodeCall(
+                CCTPBridgeActions.bridgeUSDC,
+                (
+                    address(0xBd3fa81B58Ba92a82136038B25aDec7066af3155),
+                    2e6,
+                    6,
+                    bytes32(uint256(uint160(0xa11ce))),
+                    usdc_(1)
+                )
+            ),
+            "calldata is CCTPBridgeActions.bridgeUSDC(address(0xBd3fa81B58Ba92a82136038B25aDec7066af3155), 2e6, 6, bytes32(uint256(uint160(0xa11ce))), usdc_(1)));"
+        );
+        assertEq(
+            result.quarkOperations[0].expiry, BLOCK_TIMESTAMP + 7 days, "expiry is current blockTimestamp + 7 days"
+        );
+
+        // second operation
+        assertEq(
+            result.quarkOperations[1].scriptAddress,
+            address(
+                uint160(
+                    uint256(
+                        keccak256(
+                            abi.encodePacked(
+                                bytes1(0xff),
+                                /* codeJar address */
+                                address(CodeJarHelper.CODE_JAR_ADDRESS),
+                                uint256(0),
+                                /* script bytecode */
+                                keccak256(type(MorphoVaultActions).creationCode)
+                            )
+                        )
+                    )
+                )
+            ),
```

Comment:
`CodeJar.getCodeAddress`

Reply:
yeah we should use this function instead.... 
oh well looks like some old test cases are still using this complex wrapping loll...  copied from old test cases, and this assertion happened to work as intended... so didn't catch it 😅